### DISCUSSION TITLE
Is there the ability to link to the activity that resulted in the data being generated?

### DISCUSSION THREAD

**Discussion by kheerand** [ternaustralia/ontology_tern#64] [Q&A]
Is it possible to link data to an activity that caused the data to be generated?  For example, the Regional Landcare Program or the Monitoring and Evaluation Network project will generated many sets of data.  Is it possible, in the standard, to retain the association with these activities and the data?  

If not, what needs to be done to extend the standard to be able to do so?

**Comment by dr-shorthair** [comment]
This reads like a standard 'provenance' question. 
So look at https://www.w3.org/TR/prov-o/#description 
If 'data' is a subclass of 'Entity' then it is simply https://www.w3.org/TR/prov-o/#wasGeneratedBy
If you want to associate a whole data-collection campaign or activity with some funding cycle etc, then  
https://www.w3.org/TR/prov-o/#wasInformedBy
